VCFv4.1.tex

```diff
@@ -157,7 +157,7 @@ \subsubsection{Fixed fields}
 There are 8 fixed fields per record. All data lines are tab-delimited. In all cases, missing values are specified with a dot (`.'). Fixed fields are:
 
 \begin{enumerate}
-  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
+  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. (String, no white-space permitted, Required).
```

Comment:
```suggestion
  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf.\ the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. (String, no white-space permitted, Required).
```

Reply:
Notwithstanding the `cf.` on a nearby otherwise unchanged line in the 4.3 document, in this I followed my plan in PR #391:  In VCF 4.1 and 4.2 I fixed only actual errors — limiting formatting improvements to the 4.3 document. YMMV, and the VCF maintainers might wish to apply such improvements to all three documents.

Reply:
This unrelated formatting improvement is now PR #411.